### PR TITLE
Interface for FRU VPD collection

### PR DESCRIPTION
**Pull request by PriyangaRamasamy**
This commit implements com.ibm.VPD.Collection
interface which holds CollectionState property
to describe the status of FRU VPD collection.

### CODE REVIEW

**Comment by PriyangaRamasamy** [comment]
jenkins run tests please

**Review by SunnySrivastava1984 — APPROVED** [review]

**Review by spinler — APPROVED** [review]
Will this ever go upstream?

**Review by jinuthomas — APPROVED** [review]

File: yaml/com/ibm/VPD/Collection.interface.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+description: >
+    Implement to provide VPD collection state management.
+
+properties:
+    - name: CollectionStatus
+      type: enum[self.Status]
+      description: >
+          Property which tells the current status of FRU VPD collection.
```
Comment (branupama) [inline yaml/com/ibm/VPD/Collection.interface.yaml:8]: is it should be just "VPD collection" or "FRU's VPD collection" ?

File: yaml/com/ibm/VPD/Collection.interface.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+enumerations:
+    - name: Status
+      description: >
+          The current status of FRU VPD collection.
```
Comment (branupama) [inline yaml/com/ibm/VPD/Collection.interface.yaml:13]: same as above ?

**Review by branupama — CHANGES_REQUESTED** [review]

File: yaml/com/ibm/VPD/Collection.interface.yaml (hunk 1 above)
Comment (PriyangaRamasamy) [reply yaml/com/ibm/VPD/Collection.interface.yaml:8]: it gives same meaning.

Comment (branupama) [reply yaml/com/ibm/VPD/Collection.interface.yaml:8]: I thought this is for whole VPD collection status of system

**Review by branupama — APPROVED** [review]

**Review by spinler — APPROVED** [review]

**Pull request merged**